apps/condo/migrations/20220722153024-0154_organizationemployee_unique_user_and_organization.js

```diff
@@ -0,0 +1,26 @@
+// auto generated by kmigrator
+// KMIGRATOR:0154_organizationemployee_unique_user_and_organization:IyBHZW5lcmF0ZWQgYnkgRGphbmdvIDQuMCBvbiAyMDIyLTA3LTIyIDEwOjMwCgpmcm9tIGRqYW5nby5kYiBpbXBvcnQgbWlncmF0aW9ucywgbW9kZWxzCgoKY2xhc3MgTWlncmF0aW9uKG1pZ3JhdGlvbnMuTWlncmF0aW9uKToKCiAgICBkZXBlbmRlbmNpZXMgPSBbCiAgICAgICAgKCdfZGphbmdvX3NjaGVtYScsICcwMTUzX3JlbW92ZV90aWNrZXRfcGxhY2VjbGFzc2lmaWVyX2FuZF9tb3JlJyksCiAgICBdCgogICAgb3BlcmF0aW9ucyA9IFsKICAgICAgICBtaWdyYXRpb25zLkFkZENvbnN0cmFpbnQoCiAgICAgICAgICAgIG1vZGVsX25hbWU9J29yZ2FuaXphdGlvbmVtcGxveWVlJywKICAgICAgICAgICAgY29uc3RyYWludD1tb2RlbHMuVW5pcXVlQ29uc3RyYWludChjb25kaXRpb249bW9kZWxzLlEoKCdkZWxldGVkQXRfX2lzbnVsbCcsIFRydWUpKSwgZmllbGRzPSgndXNlcicsICdvcmdhbml6YXRpb24nKSwgbmFtZT0ndW5pcXVlX3VzZXJfYW5kX29yZ2FuaXphdGlvbicpLAogICAgICAgICksCiAgICBdCg==
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+--
+-- Create constraint unique_user_and_organization on model organizationemployee
+--
+CREATE UNIQUE INDEX "unique_user_and_organization" ON "OrganizationEmployee" ("user", "organization") WHERE "deletedAt" IS NULL;
```

Comment:
please make sure there are no duplicates before applying this to production database.